### PR TITLE
2: Rename Clients DLLs

### PR DESCRIPTION
**Pull request by MiYanni**
This does the Common changes for https://github.com/Azure/azure-powershell/issues/7374.

Only changed the DLL names and package names for Clients. This does not include Storage (does include Storage.Management).

### CODE REVIEW

**Comment by MiYanni** [comment]
sign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-common-sign/232/

**Comment by MiYanni** [comment]
sign: https://azuresdkci.westus2.cloudapp.azure.com/view/PowerShell/job/ps-common-sign/233/

File: src/Common/Properties/Resources.resx [hunk 1]
```diff
@@ -1473,8 +1473,8 @@ The file needs to be a PowerShell script (.ps1 or .psm1).</value>
 The data is anonymous and does not include commandline argument values.
 The data is collected by Microsoft.
 
-Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable data collection: PS &gt; Enable-AzureRmDataCollection.</value>
+Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```
Comment (markcowl) [inline src/Common/Properties/Resources.resx]: Looks like this needs to be updated to match the previous PR

File: src/ResourceManager/Properties/Resources.resx [hunk 2]
```diff
@@ -140,8 +140,8 @@
 The data is anonymous and does not include commandline argument values.
 The data is collected by Microsoft.
 
-Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Profile module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
-Use the Enable-AzureRmDataCollection cmdlet to turn the feature On. The cmdlet can be found in the AzureRM.Profile module.  To enable data collection: PS &gt; Enable-AzureRmDataCollection.</value>
+Use the Disable-AzureRmDataCollection cmdlet to turn the feature Off. The cmdlet can be found in the AzureRM.Accounts module.  To disable data collection: PS &gt; Disable-AzureRmDataCollection.
```
Comment (markcowl) [inline src/ResourceManager/Properties/Resources.resx]: Same comment

File: src/Common/Properties/Resources.resx (hunk 1 above)
Comment (MiYanni) [reply src/Common/Properties/Resources.resx]: Fixed.

File: src/ResourceManager/Properties/Resources.resx (hunk 2 above)
Comment (MiYanni) [reply src/ResourceManager/Properties/Resources.resx]: Fixed.

**Pull request merged**